source/_docs/modules-plugins-known-issues.md

```diff
@@ -516,6 +516,13 @@ Alternative plugins that have an XML sitemap feature that works well on the plat
 **Solution**: [Upgrade your site's PHP version](/docs/php-versions) to 5.5, 5.6, or 7.0.
 <hr>
 
+### [Unbounce Landing Pages](https://wordpress.org/plugins/unbounce/){.external}
+**Issue**: Click to call conversions aren't tracking even if the pages are not cached.
+
+**Solution**: Workaround is to get their pages to work with the domain by setting up a CNAME and bypassing the whole WordPress route entirely. Setting-up a subdomain for the root domain and set up the CNAME pointing to unbouncepages.com as a solution outlined [here](https://documentation.unbounce.com/hc/en-us/articles/203661044-Connecting-Your-Domain-to-Unbounce){.external}.
```

Comment:
@carl-alberto I can't edit this yet, there's a lot of stuff here that needs to be better clarified.

>Workaround is to get their pages to work with the domain by setting up a CNAME and bypassing the whole WordPress route entirely.

"Workaround" Meaning how to get conversions  tracked?

"their pages" Who, and what pages?

Work with the domain?

I could go on, but in general this solution needs to be reworked to make sense to someone without the context of being in your head.